examples/databinding/.bazelrc

```diff
@@ -0,0 +1,13 @@
+# Flags needed while the Android rules are being migrated to Starlark.
+common --experimental_google_legacy_api
+common --experimental_enable_android_migration_apis
+common --android_sdk=@androidsdk//:sdk
```

Comment:
Lines 2-4 should no longer be necessary (in fact they should also be deleted from the basic example app, if that's where you copied these from)